documentation/translations/translating.rst

```diff
@@ -234,14 +234,16 @@ is provided below:
            print(kw, ":", keywords[kw])
 
 
+.. _transifex-use:
+
 Transifex
 =========
 
 .. important::
 
    There are many translations in the `python-doc organization on Transifex <tx_>`_,
    some of which, however, are not used or do not have a coordination team.
-   Confirm this is not the case before you begin translating.
+   Confirm that a coordination team exists before you begin translation.
```

Comment:
`begin translation` doesn't read too well. Let's do either `begin translating` or `start with the translation`.

```suggestion
   Confirm that a coordination team exists before you begin translating.
```